### DISCUSSION TITLE
Is it necessary to override the build command for Next.js SSG websites?

### DISCUSSION THREAD

**Discussion by buckwhitzer** [vercel/community#1862] [Help] [answered]
### Summary

Hey everyone!

I'm about to launch my website which is based on Next.js' SSG and I wanted to ask some questions about deployment preparations. If my site is SSG should I override the build command from the default one to 'next build && next export'?

And should I leave the other build settings as their default on Vercel without overriding? Thanks a lot!

**Comment by amyegan** [comment, marked as answer]
You can use the default Next.js framework preset for this. Your website code and configuration options are what define SSG, SSR, and ISR strategies. The general rule is that `getStaticProps` is for static pages and `getServerSideProps` is for server-side rendered pages. You can even have a mix of static and server-side rendered pages for the same site

**Reply by buckwhitzer** [reply]
Thanks a lot!
